frontend/app/src/components/settings/data-security/RestoreAssetsDatabase.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <fragment>
+    <card class="mt-8">
+      <template #title>{{ $t('asset_update.restore.title') }}</template>
+      <template #subtitle>{{ $t('asset_update.restore.subtitle') }}</template>
+      <v-btn
+        depressed
+        color="primary"
+        class="mt-2"
+        @click="activateRestoreAssets()"
+      >
+        {{ $t('asset_update.restore.action') }}
+      </v-btn>
+    </card>
+    <confirm-dialog
+      :display="confirmRestore"
+      :title="$t('asset_update.restore.delete_confirmation.title')"
+      :message="$t('asset_update.restore.delete_confirmation.message')"
+      @confirm="restoreAssets"
+      @cancel="confirmRestore = false"
+    />
+    <confirm-dialog
+      :display="doubleConfirmation"
+      :title="$t('asset_update.restore.hard_restore_confirmation.title')"
+      :message="$t('asset_update.restore.hard_restore_confirmation.message')"
+      @confirm="confirmHardReset"
+      @cancel="doubleConfirmation = false"
+    />
+    <confirm-dialog
+      v-if="done"
+      single-action
+      display
+      :title="$t('asset_update.restore.success.title')"
+      :primary-action="$t('asset_update.success.ok')"
+      :message="$t('asset_update.restore.success.description')"
+      @confirm="updateComplete()"
+    />
+  </fragment>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import { mapActions } from 'vuex';
+import ConfirmDialog from '@/components/dialogs/ConfirmDialog.vue';
+import Fragment from '@/components/helper/Fragment';
+import BackendMixin from '@/mixins/backend-mixin';
+import { Severity } from '@/store/notifications/consts';
+import { notify } from '@/store/notifications/utils';
+@Component({
+  components: {
+    ConfirmDialog,
+    Fragment
+  },
+  methods: {
+    ...mapActions('session', ['logout'])
+  }
+})
+export default class RestoreAssetsDatabase extends Mixins(BackendMixin) {
```

Comment:
this is more of an FYI and it doesn't need action in the context of this PR, but in new components and I am slowly moving to composition API using https://github.com/vuejs/composition-api in order to slowly prepare to migrate to vue 3 at some point at the end of the year probably along with the v3 of Vuetify.